src/main/kotlin/com/plus/taxiapp/domain/Member.kt

```diff
@@ -0,0 +1,13 @@
+package com.plus.taxiapp.domain
+
+data class Member(
+    var id: Long? = null,
+    var name: String,
+    val defaultPaymentType: PaymentType? = null,
+    val defaultPaymentId: Long? = null,
+    val account: MutableList<Account>? = null,
+)
+
+enum class PaymentType {
+    ACCOUNT, CARD
```

Comment:
🚫 **[ktlint]** <standard:enum-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Enum entry should start on a separate line